src/app/tabs/compile-tab.js

```diff
@@ -103,7 +103,21 @@ class CompileTab extends ViewPlugin {
     })
 
     this.fileManager.events.on('currentFileChanged', (name) => {
-      this.compilerContainer.currentFile = name
+      this.fileManager.events.on('currentFileChanged', (name) => {
+        this.fileManager.getFile(name).then(data => {
+          let pragmaArr = data.match(/(pragma solidity (.+?);)/g)
+          if (pragmaArr && pragmaArr.length === 1) {
+            let pragma = pragmaArr[0].replace('pragma solidity', '').trim()
```

Comment:
I don't think this is handling the following case : `pragma solidity >=0.4.0 <0.6.0;`

Reply:
Currently, for a range in pragma, I am thinking to use default version as we are not sure which version to load. Looking for the suggestions @yann300

Reply:
i think we should select the latest available from the given range